home/.aliases

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
```

Comment:
What's the difference between this file and the bash_alias file? Seems like duplication of efforts.